src/components/BookPage.js

```diff
@@ -1,29 +1,31 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Booklist from './Booklist';
 import Form from './Form';
 
 const BookPage = () => {
-  const allbooks = [
-    {
-      id: 1,
-      title: 'A Tale of two Cities',
-      author: 'Charles Dickens',
-    },
-    {
-      id: 2,
-      title: 'In the name of the Wind',
-      author: 'Patrick Rothfuss',
-    },
-    {
-      id: 3,
-      title: 'A Song of Ice and Fire',
-      author: 'George R. R. Martin',
-    },
-  ];
+  const books = useSelector((state) => state.book);
+  // const allbooks = [
+  //   {
+  //     id: 1,
+  //     title: 'A Tale of two Cities',
+  //     author: 'Charles Dickens',
+  //   },
+  //   {
+  //     id: 2,
+  //     title: 'In the name of the Wind',
+  //     author: 'Patrick Rothfuss',
+  //   },
+  //   {
+  //     id: 3,
+  //     title: 'A Song of Ice and Fire',
+  //     author: 'George R. R. Martin',
+  //   },
+  // ];
```

Comment:
- [ ] It's a good convention to have your codebase clean and lean and therefore, it's not a good practice to have code you're not using commented on in your project. Kindly remove these commented codes 👍